changelog/new_add_restrict_on_send_for_cop_template.md

```diff
@@ -0,0 +1 @@
+* [#10743](https://github.com/rubocop/rubocop/pull/10743): Add `RESTRICT_ON_SEND` to the Cop template generated by the `new_cop` task. ([@ydah][])
```

Comment:
No change log is required because the change is an internal improvements and have no user impact. Can you remove this entry?

Reply:
I thought I saw a PR that had added `InternalAffairs` cop to the changelog, so I had added it, but it is certainly unnecessary. I deleted it.